CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## v0.0.6-beta
```

Comment:
i considered dropping the suffix entirely and going with just `v0.0.6`; any opinions on that?

Reply:
I am in favor of dropping it. being a major and minor version 0 indicates it's prerelease anyway.

Reply:
works for me 👍